controllers/operatorconfigmap_controller.go

```diff
@@ -177,6 +187,7 @@ func (c *OperatorConfigMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 //+kubebuilder:rbac:groups=console.openshift.io,resources=consoleplugins,verbs=*
 //+kubebuilder:rbac:groups=operators.coreos.com,resources=subscriptions,verbs=get;list;watch;update
 //+kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingwebhookconfigurations,verbs=get;list;update;create;watch;delete
+//+kubebuilder:rbac:groups=csi.ceph.io,resources=*,verbs=*
```

Comment:
This looks to be wider permission, we can settle on what is required?

Reply:
2 reasons I didn't do that yet, one except the mapping API we need all permissions for every other API, two not directly related, I'm waiting for names to be finalized.

Reply:
done.